qcodes/tests/dataset/test_dataset_basic.py

```diff
@@ -47,6 +47,15 @@ def test_has_attributes_after_init():
         getattr(ds, attr)
 
 
+@pytest.mark.usefixtures("experiment")
+@pytest.mark.parametrize("non_existing_run_id", (999999, 'number#42'))
```

Comment:
When using the `experiment` fixture, you get back `e` where `e = new_experiment("test-experiment", sample_name="test-sample")`. Therefore, `1` should also be a non-existing run_id. That would in some way be a good check of the fixture to use `1` instead of / in addition to `999999` and a string.